azure-pipelines/dotnet-test-cloud.ps1

```diff
@@ -0,0 +1,37 @@
+Param(
+    [string]$Configuration='Debug',
+    [string]$Agent='Local',
+    [switch]$PublishResults
+)
+
+$RepoRoot = Resolve-Path (Join-Path $PSScriptRoot '..')
```

Comment:
Similar to the comment above `Resolve-Path "$PSScriptRoot/.."` might be a more readable alternative.